tests/test-in-docker-image.sh

```diff
@@ -121,6 +123,15 @@ function test_playbook(){
 
     echo "TEST: 3 idempotence test! Same as previous (only second play in test.yml) but now grep for changed=0.*failed=0"
     ansible-playbook -i ${ANSIBLE_INVENTORY} ${ANSIBLE_PLAYBOOK_IDEMPOTENCY} ${ANSIBLE_LOG_LEVEL} --connection=local ${SUDO_OPTION} ${ANSIBLE_EXTRA_VARS} | grep -q 'changed=0.*failed=0' && (echo 'Idempotence test: pass' ) || (echo 'Idempotence test: fail' && exit 1)
+
+    echo "TEST: 4 factless! ansible-playbook -i ${ANSIBLE_INVENTORY} ${ANSIBLE_PLAYBOOK_FACTLESS} ${ANSIBLE_LOG_LEVEL} --connection=local ${SUDO_OPTION} ${ANSIBLE_EXTRA_VARS}"
+    # instead of having multiple very similar playbooks we copy them and modify them with sed
```

Comment:
If the role works without facts, how about changing the test to _only_ test with `gather_facts: False` or is there some use case that I'm missing?

Reply:
Can be done I guess. I don't know the use case though so I would be OK with keeping it like this in case it materializes in the future and running this role with facts for some reason stops working..

Reply:
Is fiiiiiiiiiiiiiiiine